lib/github_fetcher.rb

```diff
@@ -151,15 +151,34 @@ def ignored_ci_repos
     YAML.load_file(File.join(File.dirname(__FILE__), "../ignored_ci_repos.yml"))
   end
 
+  def is_ruby_app?(repo)
+    github.contents(repo, path: "Gemfile")
```

Comment:
We need to check if it's a Rails app as Brakeman is Ruby on Rails specific. Non-Rails apps like Seal don't need it.